podcasts/Onboarding/Plus/FeaturesCarousel.swift

```diff
@@ -30,17 +31,23 @@ struct FeaturesCarousel: View {
                             if cardHeights.count == tiers.count {
                                 calculatedCardHeight = cardHeights.max()
 
+                                if (calculatedCardHeight ?? 0) > (calculatedCardMaxHeight ?? 0) {
+                                    calculatedCardMaxHeight = calculatedCardHeight
+                                }
+
                                 // Reset the card heights so any view changes won't use old data
                                 cardHeights = []
                             }
                         }
                     }
                 )
+                .frame(maxHeight: calculatedCardHeight, alignment: .top)
         }
         .carouselPeekAmount(.constant(tiers.count > 1 ? ViewConstants.peekAmount : 0))
         .carouselItemSpacing(ViewConstants.spacing)
         .carouselScrollEnabled(tiers.count > 1)
-        .frame(height: calculatedCardHeight)
+        // Maintain the largest height
+        .frame(height: calculatedCardMaxHeight, alignment: .top)
```

Comment:
When calculating the height, this will always use the absolute max height we've seen which prevents the container view from resizing down when the card height changes. 

Using the `alignment` here will ensure the cards stay at the top rather than being centered. 

On the cards we use `.frame(maxHeight: calculatedCardHeight, alignment: .top)` to ensure they get properly sized, and aligned for their content size.